debug/watcher.py

```diff
@@ -85,8 +90,11 @@ def __init__(self, dic, line_adapter = None, verbose = False):
                 try:
                     raw_file_name, line = line_adapter.failback()
                 except Exception as e:
-                    raise RuntimeError("Failed to set breakpoint %s\n%s" % (
-                        cb.__name__, e
+                    # This location format is compatible with PyDev (Eclipse)
+                    # and same as one used in standard exception tracebacks.
```

Comment:
Standard exception is `File "/home/user/qdt/debug/watcher.py", line 97, in __init__`

Reply:
So, it will be better if it is changed to
```
                    cb_file, cb_line = cb_loc(cb)
                    loc_str = '  File "%s", line %d, in %s' % (cb_file, cb_line,
                        cb.__name__
                    )
                    raise RuntimeError("Failed to set breakpoint\n%s\n%s" % (
                        loc_str, e
                    ))
```

Reply:
1) It is not a traceback.
2) the line is not `in` the function.